minetest/minetest
# DevTest: Add item meta editor

Adds an Item Meta Editor to DevTest.

This is a tool which allows you to edit the metadata of items. It's like the Node Meta Editor, but for items.

## How to test

Get the Item Meta Editor and put an item in the next slot. Punch or place with the Item Meta Editor and an editor window appears. Here you can get and set metadata.

The Item Meta Editor works like the Node Meta Editor.

Try out setting `color`, `palette_index`, `description`, etc.
